physical/inmem/cache_test.go

```diff
@@ -63,268 +64,3 @@ func TestCache_Purge(t *testing.T) {
 		t.Fatalf("should not have key")
 	}
 }
-
```

Comment:
Should we add test cases for when cache.SetEnabled(false) to make sure operations are not getting/modifying from cached values?

Reply:
I can add such a test, good idea.

Reply:
Done!